mmengine/hooks/checkpoint_hook.py

```diff
@@ -460,6 +466,20 @@ def _save_checkpoint_with_step(self, runner, step, meta):
         with open(save_file, 'w') as f:
             f.write(self.last_ckpt)  # type: ignore
 
+        # in some environments, `os.symlink` is not supported, then set
+        # 'create_symlink' to False.
+        if self.create_symlink:
+            dst_file = osp.join(runner.work_dir, 'latest.pth')
+            try:
+                symlink(ckpt_filename, dst_file)
+            except SystemError:
+                print_log(
+                    'create_symlink is set as False because creating symbolic '
+                    f'link is not allowed in {self.file_client.name}',
+                    logger='current',
+                    level=logging.WARNING)
```

Comment:
```suggestion
            except SystemError as err:
                print_log(
                    'create_symlink is set as False because failed to create a symbolic '
                    f'link for {err}', logger='current', level=logging.WARNING)
```